app/assets/javascripts/app/discussions/templates/discussion.html.slim

```diff
@@ -0,0 +1,8 @@
+.TopBarLayout
+  .TopBarLayout-header
+    md-toolbar.md-padding.md-whiteframe-z1.md-accent.u-gamma ui-view="header" layout="row" layout-align="space-between center"
```

Comment:
Why is `u-gamma` needed here? I don't remember it being necessary on the toolbar.